packages/mux-player/README.md

```diff
@@ -225,22 +225,23 @@ Offers a way to turn off keyboard shortcuts that should not be enabled.
 The `hotkeys` attribute accepts a blocklist as a space separated string.
 
 Supported **tokens**:
-`nof`, `nok`, `nom`, `noarrowleft`, `noarrowright`, `nospace`
+`noc`, `nof`, `nok`, `nom`, `noarrowleft`, `noarrowright`, `nospace`
 
 ```html
 <mux-player playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe" hotkeys="noarrowleft noarrowright"></mux-player>
 ```
 
 #### Default hotkeys
 
-| Key   | Behavior          |
-| ----- | ----------------- |
-| Space | Toggle Playback   |
-| `k`   | Toggle Playback   |
-| `m`   | Toggle mute       |
-| `f`   | Toggle fullscreen |
-| ⬅     | Seek back 10s     |
-| ➡     | Seek forward 10s  |
+| Key     | Behavior                     |
+| ------- | ---------------------------- |
+| `Space` | Toggle Playback              |
+| `k`     | Toggle Playback              |
+| `m`     | Toggle mute                  |
+| `f`     | Toggle fullscreen            |
+| `c`     | Toggle captions or subtitles |
```

Comment:
It doesn't bother me *at all* that the orders of these are not consistent wrt docs... nope nope.

Reply:
going to randomize the order of these across files